esp/src/eclwatch/LZBrowseWidget.js

```diff
@@ -393,16 +393,25 @@ define([
 
         _onSprayXml: function(event) {
             var context = this;
-            this._spraySelectedOneAtATime("SprayXmlDropDown", "SprayXmlForm", function (request, item) {
-                lang.mixin(request, {
-                    sourceRowTag: item.targetRowTag
+            this.sprayXmlTargetRowTag = registry.byId(this.id + "sprayXmlTargetRowTag");
+            if (this.sprayXmlTargetRowTag.getValue() !== "") {
```

Comment:
(Minor) I would probably just have:
```
if (this.sprayXmlTargetRowTag.getValue()) {
```
That way it also tests for null as well as "" (Its minor as I don't think you can get a null / undefined here).